src/openrct2/drawing/Text.cpp

```diff
@@ -156,6 +156,12 @@ void gfx_draw_string_left(rct_drawpixelinfo* dpi, rct_string_id format, void* ar
     DrawTextCompat(dpi, x, y, format, args, colour, TextAlignment::LEFT);
```

Comment:
Make this call the new function instead, so we don't duplicate code
```suggestion
    gfx_draw_string_left(dpi, format, args, colour, { x, y});
```